tests/configure/test_cli.py

```diff
@@ -120,3 +121,11 @@ def test_configure_cli_jobs_api_version_password():
     assert get_config().host == TEST_HOST
     assert get_config().username == TEST_USER
     assert get_config().password == TEST_PASSWORD
+
+
+def test_configure_cli_oauth():
+    runner = CliRunner()
+    runner.invoke(cli.configure_cli, ['--oauth'],
+                  input=(TEST_HOST + '\n' + TEST_SCOPE + '\n'))
+    # This is expected to error because the host doesn't exist and is not
```

Comment:
Do we need to catch the exception here so as to verify the exception is thrown as expected?